### PR TITLE
Team 5 Added click sounds, updated tower descriptions and their fonts

### PR DESCRIPTION
**Pull request by SonjaMcNeilly**

### CODE REVIEW

File: source/core/src/main/com/csse3200/game/screens/TowerType.java
Comment (ThivanW) [inline source/core/src/main/com/csse3200/game/screens/TowerType.java:1]: Youve changed the costs of the tnt tower droid tower and weapon tower back to their original costs which is not what we want

**Review by ThivanW — CHANGES_REQUESTED** [review]
Just need to keep the costs of the towers as what they need to be and then Im happy for it to be merged

File: source/core/src/main/com/csse3200/game/screens/TowerType.java
Comment (SonjaMcNeilly) [reply source/core/src/main/com/csse3200/game/screens/TowerType.java:1]: Ohhh sorry about that, ill update that now in alignment to what is in main

Comment (The-AhmadAA) [inline source/core/src/main/com/csse3200/game/screens/TowerType.java:1]: Seconding Thivan's comment, you may need to pull main into your branch before merging

**Review by The-AhmadAA — CHANGES_REQUESTED** [review]

**Review by ThivanW — APPROVED** [review]

**Review by The-AhmadAA — APPROVED** [review]

**Pull request merged**